src/TrendsOverview.svelte

```diff
@@ -34,10 +34,13 @@
     setSelectedCounty,
     setSelectedState,
   } from "./choropleth.js";
-  import { updateWithZipsData } from "./zip_data"
+  import { updateWithZipsData } from "./zip_data";
   import { fetchCountryMetaData } from "./metadata";
   import TimeSeries from "./TimeSeries.svelte";
   import TopQueries from "./TopQueries.svelte";
+  import { _, locale } from "svelte-i18n";
+import { i } from "mathjs";
+import { html } from "d3";
```

Comment:
Just nit pic, seems like these two imports are not used, so please remove. Or if it's needed, add tabs to match the rest